test/test_multicohort.py

```diff
@@ -241,7 +232,8 @@ def test_multicohort(mocker: MockFixture, tmp_path):
 
     mocker.patch('cpg_workflows.metamist.Metamist.get_ped_entries', mock_get_pedigree)
     mocker.patch('cpg_workflows.metamist.Metamist.get_analyses_by_sgid', mock_get_analysis_by_sgs)
-    mocker.patch('cpg_workflows.metamist.Metamist.get_sgs_for_cohorts', mock_get_cohorts)
+    # don't patch the method location, patch where it's imported/called
+    mocker.patch('cpg_workflows.inputs.get_cohort_sgs', mock_get_cohorts)
```

Comment:
Because the method is now static, we don't mock it in the context of the Metamist instance, we mock it where it's imported